ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -677,3 +682,67 @@ def verify_storage_clients_page(self):
             page_name="client_onboarding_token_page",
         )
         return storage_client_obj
+
+    def get_est_days_from_ui(self):
+        """
+        Get the value of 'Estimated days until full' from the UI
+
+        Returns:
+            int: Estimated days until full from UI
+
+        """
+        from ocs_ci.ocs.ui.page_objects.block_and_file import BlockAndFile
+
+        collected_tpl_of_days_and_avg = (
+            BlockAndFile().odf_storagesystems_consumption_trend()
+        )
+        est_days = re.search(r"\d+", collected_tpl_of_days_and_avg[0]).group()
+        logger.info(f"'Estimated days until full' from the UI : {est_days}")
+        return int(est_days)
+
+    def get_avg_consumption_from_ui(self):
```

Comment:
Please move `get_avg_consumption_from_ui` to BlockAndFile similarly to odf_storagesystems_consumption_trend

Reply:
this way you will not require imports as well